alfred-telemetry-solr/README.md

```diff
@@ -78,39 +78,41 @@ Following custom metrics have been implemented.
 
 At the moment following metrics are included, for each core tracked by solr:
 
-| Name                                                                   |
-| -----------------------------------------------------------------------|
-| alfresco.nodes{core=<core>,feature="Alfresco Nodes in Index"}          |
-| alfresco.nodes{core=<core>,feature="Alfresco Transactions in Index"}   |
-| alfresco.nodes{core=<core>,feature="Alfresco Stated in Index"}         |
-| alfresco.nodes{core=<core>,feature="Alfresco Unindexed Nodes"}         |
-| alfresco.nodes{core=<core>,feature="Alfresco Error Nodes in Index"}    |
-| alfresco.acls{core=<core>,feature="Alfresco Acls in Index"}            |
-| alfresco.acls{core=<core>,feature="Alfresco Acl Transactions in Index"}|
+| Name                                                       |
+| -----------------------------------------------------------|
+| alfresco.nodes{core=\<core\>,feature="Nodes"}                |
+| alfresco.nodes{core=\<core\>,feature="Unindexed Nodes"}      |
+| alfresco.nodes{core=\<core\>,feature="Error Nodes"}          |
+| alfresco.acls{core=\<core\>,feature="Acls"}                  |
+| alfresco.states{core=\<core\>}                               |
```

Comment:
what are those states ?

Reply:
They are documents in solr which keep the states of the trakers.
There should be 2 such documents.